### PR TITLE
feat(integrations) Add page for new Slack integration

### PR DESCRIPTION
**Pull request by jessla**
## Rendered 
sentry.io docs - updated 3/22/18

TODO
- [x] update global integrations image once "configure" button is added 
- [x] check all images before merging 
- [x] add content on configuring a new rule from scratch 
- [x] add note that this is only available for early adopters 

cc @saragilford @EvanPurkhiser

### CODE REVIEW

**Comment by EvanPurkhiser** [comment]
`docs/integrations/img/slack-alert-message.png` Has a black border on the left, seems unintentional.

**Comment by jessla** [comment]
@EvanPurkhiser fixed!

File: docs/integrations/slack.rst [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+.. note:: The new Slack integration is only available for Sentry's early adopters -- you can enable early adopters in your **Organization Settings** under **General**.
+
+Sentry's new Slack integration creates workflows for your team, instead of a notification stream. Now you can triage, resolve, and ignore issues directly from Slack.
+
+Configure Slack
+---------------
+
+You can authorize the Slack integration globally by clicking on **All Integrations** under your **Project settings**.
+
```
[the remark this answers is not in the document]
Comment (jessla) [reply docs/integrations/slack.rst]: yup will do

[the remark this answers is not in the document]
Comment (jessla) [reply docs/integrations/slack.rst]: updated the rendered view in the OP to reflect this

**Comment by jessla** [comment]
@EvanPurkhiser do you mind giving this another 👀 and approving if everything looks good? trying to merge today (will follow up with on-prem docs in another pr) 🚀

**Pull request merged**